presto-docs/src/main/sphinx/release/release-330.rst

```diff
@@ -0,0 +1,74 @@
+=========================
+Release 330 (XX Feb 2020)
+=========================
+
+General Changes
+---------------
+
+* Fix incorrect behavior of :func:`format` for ``char`` values. Previously, the function
+  did not preserve trailing whitespace of the value being formatted. (:issue:`2629`)
+* Fix query failures when dynamic filtering is on for certain multi-level joins. (:issue:`2659`)
+* Fix failure of ``SELECT`` queries when accessing ``information_schema`` schema tables with empty
+  value used in predicate (like ``SELECT * FROM catalog.information_schema.tables WHERE table_name=''``).
+  (:issue:`2575`)
+* Fix query failure when :doc:`/sql/execute` is used with an expression containing a function call. (:issue:`2675`)
+* Fix dynamic filtering predicate inference for certain co-located joins. (:issue:`2685`)
+* Fix failure in ``SHOW CATALOGS`` when the user does not have permissions to see any catalogs. (:issue:`2593`)
+* Improve query performance for some join queries when :doc:`cost based optimizations</optimizer/cost-based-optimizations>`
```

Comment:
I'd remove `cost based optimizations` and would use bare link just as we do in other places